public/emojify.js

```diff
@@ -1,24 +1,163 @@
+    'ledger', 'red_book', 'green_book', 'blue_book', 'orange_book', 'books', 'book', 'bookmark', 'link', 'paperclip',
+    'carpenter_square', 'ruler', 'push_pin', 'pin', 'scissors', 'memo', 'pencil', 'search', 'privacy', 'secure',
+    'locked', 'unlocked', 'heart', 'yellow_heart', 'green_heart', 'blue_heart', 'purple_heart', 'broken_heart',
+    'two_hearts', 'revolving_hearts', 'heartbeat', 'heart_pulse', 'sparkling_heart', 'cupid', 'gift_heart', 'heart_box',
+    'ophiuchus', 'aries', 'taurus', 'gemini', 'cancer', 'leo', 'virgo', 'libra', 'scorpius', 'sagittarius', 'capricorn',
+    'aquarius', 'pisces', 'id', 'phone_off', 'vibration_mode', 'eight_pointed_star', 'vs', 'white_flower', 'a', 'b',
+    'ab', 'cl', 'o', 'sos', 'cross_mark', 'circle', 'no_entry', 'name_badge', 'prohibited', '100', 'anger',
+    'hot_springs', 'no_pedestrians', 'do_not_litter', 'no_bicycles', 'non-potable_water', 'underage', 'no_phones',
+    'no_smoking', 'exclamation', 'grey_exclamation', 'question', 'grey_question', 'bangbang', 'interrobang',
+    'low_brightness', 'brightness', 'part_alternation', 'warning', 'children_crossing', 'trident', 'beginner',
+    'recycle', 'check', 'stock_market', 'sparkle', 'eight_spoked_asterisk', 'x', 'www', 'cute', 'metro', 'cyclone',
+    'zzz', 'atm', 'wc', 'accessible', 'parking', 'passport_control', 'customs', 'baggage_claim', 'locker', 'mens',
+    'womens', 'baby_change_station', 'restroom', 'put_litter_in_its_place', 'cinema', 'cell_reception', 'symbols',
+    'info', 'abc', 'abcd', 'capital_abcd', 'ng', 'squared_ok', 'squared_up', 'cool', 'new', 'free', 'ten', '1234',
+    'play', 'fast_forward', 'rewind', 'double_up', 'double_down', 'play_reverse', 'upvote', 'downvote', 'right', 'left',
+    'up', 'down', 'upper_right', 'lower_right', 'lower_left', 'upper_left', 'up_down', 'left_right', 'forward', 'reply',
+    'heading_up', 'heading_down', 'shuffle', 'repeat', 'repeat_one', 'counterclockwise', 'clockwise', 'music',
+    'musical_notes', 'plus', 'minus', 'division', 'multiplication', 'dollars', 'exchange', 'tm', 'wavy_dash', 'loop',
+    'double_loop', 'end', 'back', 'on', 'top', 'soon', 'check_mark', 'checkbox', 'radio_button', 'white_circle',
+    'black_circle', 'red_circle', 'blue_circle', 'red_triangle_up', 'red_triangle_down', 'small_orange_diamond',
+    'small_blue_diamond', 'large_orange_diamond', 'large_blue_diamond', 'black_and_white_square',
+    'white_and_black_square', 'black_small_square', 'white_small_square', 'black_medium_small_square',
+    'white_medium_small_square', 'black_medium_square', 'white_medium_square', 'black_large_square',
+    'white_large_square', 'speaker', 'mute', 'softer', 'louder', 'notifications', 'mute_notifications', 'megaphone',
+    'loudspeaker', 'umm', 'thought', 'spades', 'clubs', 'hearts', 'diamonds', 'joker', 'playing_cards', 'mahjong',
+    'time', 'checkered_flag', 'triangular_flag', 'crossed_flags'];
```

Comment:
For all these I created #143. So, I don't think you should format all this. Please remove these changes.

Reply:
Sorry for late replying, I removed formatting and rebased my branch.